sdk/identity/Cargo.toml

```diff
@@ -47,6 +47,7 @@ enable_reqwest_rustls = [
 development = []
 test_e2e = []
 client_certificate = ["openssl"]
+vendored-openssl = ["openssl/vendored"]
```

Comment:
All other features use `_`.

Reply:
i didn't put a lot of thought into this. Almost every other library i have used uses vendored-openssl for this intent. I will change it to vendored_openssl for now to follow convention of this library